administrator/components/com_content/models/forms/article.xml

```diff
@@ -672,6 +672,15 @@
 			description="COM_CONTENT_FIELD_IMAGE_CAPTION_DESC"
 			size="20"/>
 	</fields>
+	<fields name="menulink" label="COM_CONTENT_MENU_LINK_OPTIONS">
+		<field
+				name="title"
+				type="text"
+				label="COM_CONTENT_ITEM_FIELD_TITLE_LABEL"
+				description="COM_CONTENT_ITEM_FIELD_TITLE_DESC"
+				class="input-xxlarge input-large-text"
+				size="40"/>
```

Comment:
@shubhamnba2009 phpcs is not validating xml files ;)

http://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md